src/App.jsx

```diff
@@ -3,34 +3,48 @@ import React, { useState } from 'react';
 import Page from './Page';
 
 function App() {
-  const [todos, setTodos] = useState([]);
-  const [inputValue, setInputValue] = useState('');
+  const [todos, setTodos] = useState({
+    inputValue: '',
+    todo: [],
+  });
```

Comment:
```suggestion
  const [state, setState] = useState({
    inputValue: '',
    todos: [],
  });
```
배열 형태의 변수명의 경우 가능하면 복수형으로 처리하면 어떨까요?